src/interpreter.rs

```diff
@@ -139,21 +140,32 @@ impl<'a, 'tcx: 'a, 'arena> Interpreter<'a, 'tcx, 'arena> {
                 struct Instance<'tcx>(DefId, &'tcx Substs<'tcx>);
                 impl<'tcx> fmt::Display for Instance<'tcx> {
                     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-                        ppaux::parameterized(f, self.1, self.0, ppaux::Ns::Value, &[],
-                            |tcx| tcx.lookup_item_type(self.0).generics)
+                        ppaux::parameterized(f,
+                                             self.1,
+                                             self.0,
+                                             ppaux::Ns::Value,
+                                             &[],
+                                             |tcx| tcx.lookup_item_type(self.0).generics)
```

Comment:
This sticks out to me as a good example of why I don't like alignment. If the code were already a little further right, or `ppaux::parameterized` had a slightly longer name, it would have to have also line-wrapped the body of this closure.

Plus, it made 2 lines of code into 6. Vertical height matters to me when editing (amount of code I can see at once).